plugins-base/XSFeatureNTP.py

```diff
@@ -104,15 +297,23 @@ def HandleKeyADD(self, inKey):
         if inKey == 'KEY_ENTER':
             inputValues = pane.GetFieldValues()
             Layout.Inst().PopDialogue()
+            Layout.Inst().TransientBanner(Lang("Adding %s NTP Server..." % inputValues['name']))
             try:
                 IPUtils.AssertValidNetworkName(inputValues['name'])
+
                 data=Data.Inst()
+                if data.ntp.method("") == "Default":
+                    data.NTPServersSet([server for server in data.ntp.servers([]) if not inDefaultNTPDomains(server)])
```

Comment:
I don't understand this section 100% clearly: It is in HandleKeyADD to Add NTP servers, AFAICS.

When `data.ntp.method("") == "Default":`, `data.NTPServersSet(` is called with the return values (NTP servers) from `data.ntp.servers([])` that are not in the NTP default domains, correct?

I've seen that `"Default"` means that the default NTP servers shall be configured, so I'm a bit at a loss what this last line is used for, thanks!

Reply:
"Default" means to use the initial/default NTP servers ([0-3].centos.pool.ntp.org)
"Manual" means to use servers specified by the user
In the case where a user is in "Default" mode but switches to "Manual" mode, as a convenience we remove the default servers from the list
In theory this could backfire if the user manually adds the default servers to the list, then adding a new default server could remove their existing ones
The alternative here is to not remove any servers from the list when a user switches from "Default" to "Manual" mode
What do you think?